[2022-04-24 01:21:28,962] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.delete_dataset_task manual__2022-04-24T01:21:04.443060+00:00 [queued]>
[2022-04-24 01:21:28,995] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.delete_dataset_task manual__2022-04-24T01:21:04.443060+00:00 [queued]>
[2022-04-24 01:21:28,997] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 01:21:28,998] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-04-24 01:21:28,999] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 01:21:29,022] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): delete_dataset_task> on 2022-04-24 01:21:04.443060+00:00
[2022-04-24 01:21:29,036] {standard_task_runner.py:52} INFO - Started process 3831 to run task
[2022-04-24 01:21:29,051] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'local_2_gcs_2_bq_dag', 'delete_dataset_task', 'manual__2022-04-24T01:21:04.443060+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/download_2_gcs_dag_V2.py', '--cfg-path', '/tmp/tmp8dryuan7', '--error-file', '/tmp/tmpcz37gd5z']
[2022-04-24 01:21:29,054] {standard_task_runner.py:77} INFO - Job 36: Subtask delete_dataset_task
[2022-04-24 01:21:29,208] {logging_mixin.py:109} INFO - Running <TaskInstance: local_2_gcs_2_bq_dag.delete_dataset_task manual__2022-04-24T01:21:04.443060+00:00 [running]> on host 65283f82c270
[2022-04-24 01:21:29,317] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_2_gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=delete_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-24T01:21:04.443060+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-24T01:21:04.443060+00:00
[2022-04-24 01:21:29,321] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-24 01:21:29,324] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/u6pd-qa9d.csv /opt/***/u6pd-qa9d.parquet']
[2022-04-24 01:21:29,350] {subprocess.py:85} INFO - Output:
[2022-04-24 01:21:29,363] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-24 01:21:29,439] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=local_2_gcs_2_bq_dag, task_id=delete_dataset_task, execution_date=20220424T012104, start_date=20220424T012128, end_date=20220424T012129
[2022-04-24 01:21:29,508] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-24 01:21:29,578] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
